stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/state/messages/MessagesState.kt

```diff
@@ -22,7 +21,7 @@ public data class MessagesState(
     val isLoadingMore: Boolean = false,
     val endOfMessages: Boolean = false,
     val messageItems: List<MessageListItemState> = emptyList(),
-    val selectedMessage: Message? = null,
+    val selectedMessageState: SelectedMessageState? = null,
```

Comment:
Had to make this a sealed class. Having a separate `selectedMessageReactions: Message?` turned our to spawn a lot of boilerplate.